src/conda/env_eulerian_storm_track.yml

```diff
@@ -0,0 +1,16 @@
+name: _MDTF_eulerian_storm_track
```

Comment:
You can remove this file from the PR. Most package reqs are already in the MDTF and python3 base yamls, and we can add a couple more packages to them if needed.